### PR TITLE
vere: webslog atom tanks, and webflogs

### PR DESCRIPTION
**Pull request by Fang-**
As discussed out of band: fixes the webslog callback to handle cords correctly, fixes the flog callback, and sends flogs to the webslog callback.

### CODE REVIEW

File: pkg/urbit/vere/lord.c [hunk 1]
```diff
@@ -366,17 +366,15 @@ _lord_plea_slog(u3_lord* god_u, u3_noun dat)
...
+    return _lord_plea_foul(god_u, c3__flog, dat);
   }
 
-  c3_c* tan_c = u3r_string(tan);
+  c3_c* tan_c = u3r_string(dat);
   u3C.stderr_log_f(tan_c);
   c3_free(tan_c);
-  u3z(dat);
+
+  god_u->cb_u.slog_f(god_u->cb_u.ptr_v, 0, dat);
```
Comment (joemfb) [inline pkg/urbit/vere/lord.c:377]: I don't think we want to print the `%flog` messages twice, isn't that what happens here?

**Review by joemfb — CHANGES_REQUESTED** [review]
The slogstream fix looks good. And thanks for catching and fixing the uninitialized variable use in %flog handling.

But I'm don't think the logic change in %flog is quite right:

File: pkg/urbit/vere/lord.c (hunk 1 above)
Comment (Fang-) [reply pkg/urbit/vere/lord.c:377]: Oh, you're right, I got my callbacks confused. Looks like this indeed double-prints in the real terminal. Do we care about explicitly doing `stderr_log_f` here, or are we fine with saying "let the slog callback handle this"?

Comment (Fang-) [reply pkg/urbit/vere/lord.c:377]: It adds an extra `\n`, which isn't very nice... Probably best to just call the webslog callback directly here.

Comment (joemfb) [reply pkg/urbit/vere/lord.c]: This looks good, we can tweak it later.

**Review by joemfb — APPROVED** [review]
LGTM

**Comment by joemfb** [comment]
I had pushed a fix for the uninitialized variable to the target branch too. Should be simple to deconflict -- feel free to merge.

**Comment by Fang-** [comment]
Alternatively we can add a `_pier_on_lord_flog` so that `lord.c` doesn't have to know about the webslog callback, but maybe that's too much indirection for that small benefit.

**Pull request merged**